pyccl/halos/profiles.py

```diff
@@ -275,6 +275,120 @@ def cumul2d(self, cosmo, r_t, M, a, mass_def=None):
                                                 is_cumul2d=True)
         return s_r_t
 
+    def convergence(self, cosmo, r, M, a_lens, a_src, mass_def=None):
```

Comment:
Wouldn't it be better to rename `a_src` to `a_source`? Just like we don't say `a_ln` for `a_lens`.
@damonge @pennalima

Reply:
Done! @nikfilippas